src/main/java/mat/client/shared/MatContext.java

```diff
@@ -1522,13 +1522,18 @@ public void onSuccess(List<MeasureType> result) {
         });
     }
 
-    public void setMatOnFHIR(FeatureFlag matOnFHIR) {
-        this.matOnFHIR = matOnFHIR;
-    }
+	public void setFeatureFlags(List<FeatureFlag> featureFlagList){
+		this.featureFlagList = featureFlagList;
+	}
 
-    public FeatureFlag getMatOnFHIR() {
-        return this.matOnFHIR;
-    }
+	//returns if specific feature flag is on/off
+	public boolean getFeatureFlagStatus(String flag){
+		FeatureFlag featureFlag = this.featureFlagList.stream()
+				.filter(f -> flag.equals(f.getFlagName()))
```

Comment:
does it make sense to pre-compute a map?
or, we know for sure that the list is not going to grow